src/lensCalibration/LensCalibrationPlugin.cpp

```diff
@@ -134,22 +134,23 @@ void LensCalibrationPlugin::render(const OFX::RenderArguments &args)
 
       OfxPointI p(_inputPatternSize->getValue());
       cv::Size boardSize(p.x, p.y);
-      std::cout << "inputPatternSize: " << boardSize.width << ", " << boardSize.height << std::endl;
       EParamPatternType inputPatternType = EParamPatternType(_inputPatternType->getValue());
-      std::cout << "inputPatternType: " << int(inputPatternType) << std::endl;
-      openMVG::patternDetect::Pattern patternType = getPatternType(inputPatternType);
-      std::cout << "patternType openMVG: " << int(patternType) << std::endl;
+      openMVG::calibration::Pattern patternType = getPatternType(inputPatternType);
+
+      // Store the checker points of the found pattern
       std::vector<cv::Point2f> checkerPoints;
-      found = openMVG::patternDetect::findPattern(patternType, cvInputGrayImage, boardSize, checkerPoints);
+      found = openMVG::calibration::findPattern(patternType, cvInputGrayImage, boardSize, checkerPoints);
```

Comment:
`const int found = openMVG::calibration::findPattern(patternType, cvInputGrayImage, boardSize, checkerPoints);`
Because `found` is only used there.